### DISCUSSION TITLE
手动创建字段并渲染页面功能

### DISCUSSION THREAD

**Discussion by robin536180** [alibaba/formily#3810] [Q&A 1.x]
需求：form.createField手动添加字段重新渲染页面
https://codesandbox.io/s/dreamy-fog-mto2bl?file=/App.tsx
实际：创建出field 但是没有渲染到页面上
这种需要怎么实现比较好呢

https://github.com/alibaba/formily/issues/3808
各位大佬， 用 x-reaction怎么动态新增、更新节点树

**Comment by janryWang** [comment]
不能通过 createField 动态操作节点树，不支持
